lib/new_relic/agent/instrumentation/net.rb

```diff
@@ -15,75 +15,51 @@
     require 'new_relic/agent/http_clients/net_http_wrappers'
   end
 
+  def monkey_patched
+    defined?(::Airbrake) || 
+    defined?(::Rack::MiniProfiler) ||
+    source_location_for(Net::HTTP, "request") =~ /airbrake|profiler/i
```

Comment:
Since we are doing this source check for airbrake, shouldn't we also be doing this for rack miniprofiler?

Reply:
we are....line 20....or am I missing something subtle in this question?

Reply:
Oh yeah, you right, you right, it is totally there. lol I misread the regex and thought it was checking for "airbrake/profile", didn't realize it was |.